talon/signature/learning/featurespace.py

```diff
@@ -7,9 +7,12 @@
 applying features to them.
 """
 
+from __future__ import absolute_import
 from talon.signature.constants import (SIGNATURE_MAX_LINES,
                                        TOO_LONG_SIGNATURE_LINE)
 from talon.signature.learning.helpers import *
+from six.moves import zip
```

Comment:
Yep, it is used [here](https://github.com/umairwaheed/talon/blob/cefbcffd599bc10e6a7cfc692f33081861919474/talon/signature/learning/featurespace.py#L77). We are just replacing the `zip` function in the standard library of Python with the `zip` function in `six`.